src/main/app/src/Components/Landscape/Map/Map.tsx

```diff
@@ -75,7 +74,7 @@ const Map: React.FC<Props> = ({ setSidebarContent, setLocateFunction, setPageTit
         }
       }
     },
-    [value]
+    [setValue, value]
```

Comment:
As I understood the deps of the `useCallback()` hook, these are values that change upon which the hook is triggered (https://reactjs.org/docs/hooks-reference.html#usecallback). `setValue` never changes (does it?). 

I also tested removing it, it makes no difference for the focus function.

Reply:
Update: the deps should be values that are used within the hook. `setValue` is not used in this hook, we can remove it.